cmd/firearweave/cli/node.go

```diff
@@ -160,14 +160,14 @@ func nodeFactoryFunc(flagPrefix, kind string) func(*launcher.Runtime) (launcher.
 		blockStreamServer := blockstream.NewUnmanagedServer(blockstream.ServerOptionWithLogger(appLogger))
 		oneBlockStoreURL := mustReplaceDataDir(sfDataDir, viper.GetString("common-one-blocks-store-url"))
 		mergedBlockStoreURL := mustReplaceDataDir(sfDataDir, viper.GetString("common-merged-blocks-store-url"))
-		workingDir := mustReplaceDataDir(sfDataDir, viper.GetString("mindreader-node-working-dir"))
-		gprcListenAdrr := viper.GetString("mindreader-node-grpc-listen-addr")
-		mergeThresholdBlockAge := viper.GetString("mindreader-node-merge-threshold-block-age")
-		waitTimeForUploadOnShutdown := viper.GetDuration("mindreader-node-wait-upload-complete-on-shutdown")
-		oneBlockFileSuffix := viper.GetString("mindreader-node-one-block-suffix")
-		blocksChanCapacity := viper.GetInt("mindreader-node-blocks-chan-capacity")
-
-		mindreaderPlugin, err := getMindreaderLogPlugin(
+		workingDir := mustReplaceDataDir(sfDataDir, viper.GetString("reader-node-working-dir"))
+		gprcListenAdrr := viper.GetString("reader-node-grpc-listen-addr")
+		mergeThresholdBlockAge := viper.GetString("reader-node-merge-threshold-block-age")
+		waitTimeForUploadOnShutdown := viper.GetDuration("reader-node-wait-upload-complete-on-shutdown")
+		oneBlockFileSuffix := viper.GetString("reader-node-one-block-suffix")
```

Comment:
For `reader-node-one-block-suffix`, can you look back to other we did to ensure it's `reader-node-one-block-suffix`.